Client/src/pages/suggestionHistory/suggestionHistory.tsx

```diff
@@ -5,97 +5,89 @@ import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
 import SentimentDissatisfiedIcon from '@mui/icons-material/SentimentDissatisfied';
 import SentimentSatisfiedIcon from '@mui/icons-material/SentimentSatisfied';
 import SentimentSatisfiedAltIcon from '@mui/icons-material/SentimentSatisfiedAltOutlined';
+import { Mood } from '@/types/enums/Moods';
+import { MOOD_CATEGORIES, MoodCategory } from '@/types/enums/MoodGroup';
 
-interface Suggestion {
-  title: string;
-  content: string;
-  rank: number;
-  date: Date;
-  eventTitle: string;
+interface FlatEvent {
```

Comment:
put all the types in the common types folder, because we use it in all our project. (and do the types like their structure in the db)

Reply:
Its specific type only for this page, it isn't something that will be reusable.